app/pb_kits/playbook/pb_text_input/_text_input.scss

```diff
@@ -3,6 +3,59 @@
 @import "../tokens/spacing";
 @import "../tokens/typography";
 
+@mixin pb_text_input ($bg: $white, $bc: $border-light, $color: $text_lt_default) {
+  -webkit-appearance: none;
+  border: 1px solid $bc;
+  border-radius: $border-rad-light;
+  display: block;
+  width: 100%;
+  color: $color;
+  font-weight: bold;
+  background: $bg;
+  padding: $space_xs $space_sm;
+  transition: 0.2s all;
+  outline: none;
+  font-size: $font_small;
+  margin: .1rem 0 0;
```

Comment:
We are using `rem` and `px` here.. is there a reason for using `rem` where `px` wouldn't quite work out?

Reply:
This should be using our spacing variable with a math calculation of some sort too probably.